content/en/synthetics/settings/_index.md

```diff
@@ -17,9 +17,18 @@ further_reading:
   text: "Identify Synthetics Bots"
```

Comment:
This is an interesting one, because google already reads this as "Synthetics Settings", since it's nested under synthetics in the URL hierarchy . But I can see what you mean for internal search, so updating it for clarity there.

Reply:
[updated]